lib/widgets/creators_corner/workout_editor.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/constants/styles.dart';
+import 'package:workoutpersonalizer_frontend/models/exercise.dart';
+import 'package:workoutpersonalizer_frontend/models/pair.dart';
+import 'package:workoutpersonalizer_frontend/widgets/exercise_card.dart';
+
+class WorkoutEditor extends StatefulWidget {
+  const WorkoutEditor({Key? key}) : super(key: key);
+
+  @override
+  WorkoutEditorState createState() => WorkoutEditorState();
+}
+
+class WorkoutEditorState extends State<WorkoutEditor> {
+  List<Exercise> chosenExercises = [];
+
+  Widget writeText() {
+    String body;
+    if (chosenExercises.isEmpty) {
+      body = "Drag a video to the editor to add to a workout.";
+    } else {
+      body = "Drag a video to the editor to add to the workout. Drag to re-order the videos.";
+    }
+
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(35, 10, 0, 0),
+      child: Text(
+        body,
+        style: workoutEditorStyle,
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: const Color(0xFFFFA101),
```

Comment:
We should reference this color and fonts below from the styles.dart class. I made this change in my local branch already, so we can maybe just open a ticket for this, and I'll make the changes after I make a pr for my local branch.